src/cli/eest/commands/info.py

```diff
@@ -0,0 +1,57 @@
+"""Command to display EEST and system information."""
+
+import platform
+import subprocess
+import sys
+
+import click
+
+from config.app import AppConfig
+
+
+def run_command(command: list[str]) -> str:
+    """Run a CLI command and return its output."""
+    try:
+        result = subprocess.run(
+            command,
+            capture_output=True,
+            text=True,
+            check=True,
+        )
+        return result.stdout.strip()
+    except (subprocess.SubprocessError, FileNotFoundError):
+        return "unknown"
+
+
+def get_git_commit() -> str:
+    """Get the current git commit message and hash."""
+    git_hash = run_command(["git", "rev-parse", "--short=6", "HEAD"])
```

Comment:
Just FYI we use [gitpython](https://gitpython.readthedocs.io/en/stable/index.html) in other places to interact with git, but I think an external command here is fine!
https://github.com/ethereum/execution-spec-tests/blob/8435e43c19479d726698a3b91577fe4b1cdda006/src/ethereum_test_tools/utility/versioning.py#L5

Reply:
I changed it to use the util for consistency.